package.json

```diff
@@ -66,7 +66,7 @@
     "babel-plugin-add-module-exports": "^0.2.1",
     "babel-plugin-rewire": "^1.0.0",
     "babel-plugin-transform-object-assign": "^6.22.0",
-    "babel-plugin-transform-runtime": "^6.7.5",
+    "babel-plugin-transform-runtime": "^6.23.0",
```

Comment:
any reason we have this back, we were only using it for `Promise` polyfill and we replaced it with `es6-promises`